doc.go

```diff
@@ -267,10 +267,10 @@ data types. The columns are:
     INTEGER that         | int, int8, int16, int32, int64              | string                 | int, int8, int16,
     fits in int64        |                                   [1] , [2] |                        | int32, int64
   -------------------------------------------------------------------------------------------------------------------
-    INTEGER that doesn't | int, int8, int16, int32, int64, \*big.Int   | string                 | error
+    INTEGER that doesn't | int, int8, int16, int32, int64,  *big.Int   | string                 | error
```

Comment:
@sfc-gh-jbahk Mark and I discussed this, but can you verify that the slash is not required.

Reply:
The \ is not required in real text but I'm unsure how it would appear in the doc. Feel free to delete if there's no impact on the doc.

Reply:
Thanks! It looked fine in my local build using godoc.